lib/monmon.rb

```diff
@@ -3,10 +3,39 @@
 require 'net/http'
 require 'json'
 
-url = "https://api.exchangerate.host/latest?base=BYN"
-uri = URI(url)
-response = Net::HTTP.get(uri)
-RATES = JSON.parse(response)
+def rates
+  @rates ||= begin
+    config = Configuration.instance
+    url = "https://api.exchangerate.host/latest?base=#{config.main_currency}"
+    uri = URI(url)
+    response = Net::HTTP.get(uri)
+    JSON.parse(response)
+  end
+end
+
+class Configuration
```

Comment:
it's better to move class in its own file, so there it should be `configuration.rb` file, and you `require` it where it's needed.